plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -146,6 +146,7 @@ void chain_api_plugin::plugin_startup() {
    if (chain.account_queries_enabled()) {
       _http_plugin.add_async_api({
          CHAIN_RO_CALL_WITH_400(get_accounts_by_authorizers, 200, http_params_types::params_required),
+         CHAIN_RO_CALL(get_all_accounts, 200, http_params_types::no_params_required)
```

Comment:
Add this as an `add_api` above, not an `add_async_api`. Also this can always be available, no need to only be available when `account_queries_enabled()`

Reply:
Done